pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -441,7 +441,11 @@ def _initialize_deepspeed_train(self, model):
 
         # although we set these here, deepspeed manages the specific optimizer logic
         self.lightning_module.trainer.optimizers = [deepspeed_optimizer]
+
+        deepspeed_scheduler = model.lr_scheduler
         if deepspeed_scheduler is not None:
+            # disable deepspeed lr scheduling as lightning manages scheduling
+            model.lr_scheduler = None
```

Comment:
why this option instead of not passing it to `deepspeed.initialize` above?

Reply:
The main reason is if the user has used the config file to determine what the optimizer/scheduler should be (this is the case if you're using the standard deepspeed recipe config and want to replicate something) then we still need to use deepspeed to select the appropriate lr scheduler if any